vege/style.css

```diff
@@ -34,6 +35,157 @@ body {
 }
 
 
+
+
 /* ------------------------------- */
 /* ----- DOLE BUDE TVOJE CSS ----- */
 /* ------------------------------- */
+.container {
+    flex-direction: column;
+}
+h1 {
+    font-weight: 800;
+    font-size: 1.75rem;
+    padding: 10rem 0.5rem 0.5rem 0.5rem;
+    margin: 0;
+}
+h2 {
+    font-weight: 800;
+    font-size: 1.2rem;
+    padding: 5rem 0.5rem 0.5rem 0.5rem;
+    margin: 0;
+}
+p1 {
+    float: left;
+    padding: 0.5rem;
+    font-size: 0.8rem;
+    font-weight: 400;
+    
+}
+p2 {
+    float: right;
+    padding: 0.5rem;
+    font-size: 0.8rem;
+    font-weight: 400;
+}
+.hlavni {
+    background-image: url("images/salat.jpg");
+    background-size: 100%;
+    min-height: 200px;
+    background-repeat: no-repeat;
+    background-position: center;
+    flex-grow: 2;
+    padding: 0;
+    margin: 0;
+}
+.maly {
+    min-height: 90px;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    flex-grow: 1;
+    padding:0;
+    margin:0;
+}
+
+.recept2 {
+    background-image: url("images/chlebicek.jpg");
+}
+.recept3 {
+    background-image: url("images/cizrna.jpg");
+}
+.recept4 {
+    background-image: url("images/tacos.jpg");
+}
+.recept5 {
+    background-image: url("images/burger.jpg");
+}
+
+@media (min-width: 480px) {
+    .container {
+        display: flex;
+        flex-direction: column;
+        
+    }
+    .hlavni {
+        flex-basis:100%;
+        min-height: 250px;
+        flex-wrap: wrap;
+    }
+    .malerecepty {
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: wrap
+    }
+
+    .maly {
+        min-height: 150px;
+        flex-basis: 50%;
+    }
+        
+    p1{
+        padding: 1.25rem;
+    }
+    p2 {
+        padding: 1.25rem;
+    }
+    h1 {
+        padding: 40% 1.25rem 1.25rem 1.25rem;
+        font-size: 2rem;
+    }
+    h2 {
+        padding: 40% 1.25rem 1.25rem 1.25rem;
+        font-size: 1.5rem;
+    }
+}
+@media (min-width: 860px) {
+    .container {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .hlavni {
+        min-height: 400px;
+        flex-basis: 33.33%;
+        background-size: 100% 100%;
+    }
+
+    .malerecepty {
+        display: flex;
+        min-height: 400px;
+        flex-basis: 66.66%;
+    }
+    .maly {
+        min-height: 200px;
+        height: contain;
```

Comment:
Tady nevím, co přesně jsi tím myslela, ale tahle hodnota nefunguje (https://developer.mozilla.org/en-US/docs/Web/CSS/height), tak třeba v dokumentaci najdeš tu pravou :)

Reply:
Ahoj, dodatečně děkuji za všechny i předešlé komentáře. Jak jsem doma covidovala, tak jsem nějak zatím nezvládla odepsat a poděkovat. Tohle mi tam asi zůstalo tím, jak jsem bojovala s výškou obrázků a zkoušela různé varianty. Dost dlouho jsem měla problém, že jsem měla ten první největší hlavní recept na výšku jen na 80% - nevěděla jsem , že u background-size musí být dvě hodnoty a měla tam jen jednou 100%. Zkoušela jsem tak nějak kde co a z toho mi tam podle mě zůstalo to height: contain; u těch menších receptů :)

Reply:
Ahoj, jasný to chápu. Není zač a kdybys měla ještě nějaké dotazy, tak klidně řekni. Měj se. :)